src/transforms/ampValidation.js

```diff
@@ -54,4 +68,4 @@ function createErrorMessage(error, outputPath) {
   return msg;
 }
 
-module.exports = ampValidationTransform;
+module.exports = ampValidationTransforom;
```

Comment:
typo? `ampValidationTransforom`

Reply:
all mistakes in my life are blamed on the butterfly keyboard.